pkgs/applications/emulators/box64/default.nix

```diff
@@ -0,0 +1,68 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, python3
+}:
+
+stdenv.mkDerivation rec {
+  pname = "box64";
+  version = "0.1.8";
+
+  src = fetchFromGitHub {
+    owner = "ptitSeb";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-6k8Enbafnj19ATtgmw8W7LxtRpM3Ousj1bpZbbtq8TM=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    python3
+  ];
+
+  cmakeFlags = [
+    "-DNOGIT=1"
+    "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
```

Comment:
Why?

Reply:
because it seems to be one of the default flags. See [here](https://github.com/ptitSeb/box64/blob/main/docs/COMPILE.md#for-x86_64-linux)

Reply:
It's also set in the AUR: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=box64
But we usually don't build with debug symbols.

Reply:
Shouldn't they be stripped anyway in the fixup phase?

Reply:
Then it makes even less sense to use `CMAKE_BUILD_TYPE=RelWithDebInfo`.

Reply:
In generell I like the idea of shipping debug symbols with optimized binaries to make debugging easier. In the case of nixpkgs I agree with you: It shouldn't change anything really but it also  doesn't make sense to first include Debug symbols to have them removed later..

I did a comparison of `Release` and `RelWithDebInfo`. The size of the binary with `Release` is 4 MB, with `RelWithDebInfo` it is 3,8MB. I don't know why the size is smaller with the debug symbols, but :shrug:

In any case, I switched the to  `Release` here